assets/queries/terraform/kubernetes_pod/rbac_roles_with_read_secrets_permissions/query.rego

```diff
@@ -0,0 +1,17 @@
+package Cx
+
+CxPolicy[result] {
+	resourceTypes := ["kubernetes_role", "kubernetes_cluster_role"]
+	resource := input.document[i].resource[resourceTypes[t]][name]
+	readVerbs := ["get", "watch", "list"]
+	resource.rule[ru].resources[r] == "secrets"
+	resource.rule[ru].verbs[j] == readVerbs[k]
```

Comment:
Sorry, I forgot to confirm this review comment.
You still need to check when resource.rule is an object. My suggestion is to create an auxiliary function to handle both situations (array and object).
```suggestion
	allowsSecrets(resource.rule)
```
The auxiliary function `allowsSecrets` could be:
```
readVerbs := ["get", "watch", "list"]

allowsSecrets(rules) {
	is_array(rules)
	some r
	rule[r].resources[_] == "secrets"
	rule[r].verbs[_] == readVerbs[_]
}

allowsSecrets(rule) {
	is_object(rule)
	rule.resources[_] == "secrets"
	rule.verbs[_] == readVerbs[_]
```